FormElement.inc

```diff
@@ -59,6 +59,14 @@ class FormElement implements ArrayAccess {
   protected $index;
 
   /**
+   * Holds references to the original form default value for the form
+   * element.
+   *
+   * @var ReadOnlyProtected
```

Comment:
@matthewperry

This probably shouldn't be readonly.  We'll need to set it after [the alter](https://github.com/Islandora/islandora_xml_forms/commit/b6f7ffbcdeeafad518fbe0e332804eca2efea638#diff-b893f498b9bca6b1453c419abd533b59R88) to preserve any changes made there.

Thanks!

----
William Panting
_Developer_
*[discoverygarden inc.](http://www.discoverygarden.ca) | Managing Digital Content*